app.vue

```diff
@@ -15,4 +15,7 @@ watch(
   },
   { immediate: true }
 );
+
+const portalRuntimeConfig = usePortalRuntimeConfig();
+logger.log(`=== App version: ${portalRuntimeConfig.appVersion} ===`);
```

Comment:
Maybe add it to the bottom of the page (either in the bottom left corner, or in the bottom right after "Privacy Policy"), so that it's present on the screenshots? This can save us one hop during debugging.

Kind of similar to https://lite.zksync.io/ (see bottom right corner)

Reply:
This is the first thing I wanted to do, but then I discovered some of our forks intentionally hide version label on FE for explorer for instance. That's why I decided to log it.

Reply:
We can actually